openfe/setup/atommapping.py

```diff
@@ -1,8 +1,34 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+from dataclasses import dataclass
+from rdkit import Chem
+from typing import Dict, Union, TypeVar
+
+RDKitMol = TypeVar('RDKitMol')
+
+
+@dataclass
 class AtomMapping:
-    @classmethod
-    def from_perses(cls, perses_mapping):
-        pass
+    """Simple container with the mapping between two Molecules
 
-    def __getitem__(self, other):
-        return None
+    Attributes
+    ----------
+    mol1, mol2 : rdkit.Mol
+      the two Molecules in the mapping
+    mol1_to_mol2, mol2_to_mol1 : dict
+      maps the index of an atom in either molecule **A** or **B** to the other.
+      If this atom has no corresponding atom, None is returned.
 
+    """
+    mol1: RDKitMol
+    mol2: RDKitMol
+    mol1_to_mol2: Dict[int, Union[int, None]]
```

Comment:
```suggestion
    mol1_to_mol2: Dict[int, int]
```

Suggesting and accepting suggestion at suggestion of @richardjgowers